blueprints/app/files/package.json

```diff
@@ -89,17 +89,19 @@
     "ember-page-title": "^8.2.0",
     "ember-qunit": "^8.0.2",
     "ember-resolver": "^11.0.1",
-    "ember-source": "~5.7.0-beta.1",
+    "ember-source": "~5.7.0-beta.1<% if (templateTag) { %>",
+    "ember-template-imports": "^4.1.0<% } %>",
```

Comment:
I don't think we can include `ember-template-imports`, specifically, until this is resolved: https://github.com/ember-template-imports/ember-template-imports/issues/228

Reply:
resolved!